webapp-django/crashstats/api/views.py

```diff
@@ -120,8 +120,6 @@ def clean(self):
 
 # Names of models we don't want to serve at all
 BLACKLIST = (
-    # only used for doing posts
-    'Releases',
```

Comment:
We want to keep this. It's related to the releases API. We only want to remove the "add release" form from the admin panel.